src/wagtail_personalisation/templates/modeladmin/wagtail_personalisation/segment/dashboard.html

```diff
@@ -1,5 +1,5 @@
 {% extends "modeladmin/wagtail_personalisation/segment/base.html" %}
-{% load i18n l10n static modeladmin_tags wagtailadmin_tags wagtail_personalisation_filters %}
+{% load i18n l10n static modeladmin_tags wagtail_personalisation_filters %}
```

Comment:
My thinking was that if the fork for 5.1 support is backwards compatible and doesn't require too many changes right now to work with 4.1, then in the long run it can save a lot of time in having to update and check this repo again. 
Only having to revert the one set of changes regarding the icons seemed minimal enough to me to still keep using this version. 
I'm of course open to changing my mind on this if you think it's more risky or anything!